templates/partners/channel-and-reseller.html

```diff
@@ -6,97 +6,113 @@
 
 {% block meta_description %}Canonical works with channel and reseller partners around the world to ensure that Canonical offerings are available for their customers.{% endblock %}
 
+{% set is_paper = True %}
+
 {% block content %}
-<section class="p-strip--image u-no-padding--top is-dark">
-  <div class="p-strip--suru-top">
-    <div class="row">
-      <div class="col-8">
-        <h1>Channel&sol;Reseller Programme</h1>
+
+{% with parent_name="Partners", parent_href="/partners", title="Channel / Reseller Programme" %}
+  {% include '/partial/_breadcrumbs.html' %}
+{% endwith %}
+
+<section class="p-section">
+  <div class="row--50-50">
+    <div class="col">
+      <h1>Channel&sol;Reseller Programme</h1>
+    </div>
+    <div class="col">
+      <div class="p-block">
         <p>Canonical works with channel and reseller partners around the world to ensure that Canonical offerings are available for their customers. Additionally these partners can both resell services and value-added offerings around our technology enabling them to grow their business.</p>
         <p>Canonical is an industry leader in OpenStack private clouds, the reference platform for all major public clouds, and the open source software operating system that runs from the desktop and all internet connected things. Our success helps our partners build a business around our technology.</p>
-        <a href="https://partner-portal.canonical.com" class="p-button"><span>Login to the partner portal</span></a>
-      </div>
-      <div class="col-4 u-hide--small u-hide--medium u-align--center u-vertically-center">
-        <img src="https://assets.ubuntu.com/v1/80472358-Canonical-reseller-icon.svg" alt="">
       </div>
+      <a href="https://partner-portal.canonical.com" class="p-button--positive"><span>Login to the partner portal</span></a>
+      <a href="/partners/become-a-partner" class="p-button">Become a partner</a>
     </div>
   </div>
 </section>
 
-<section class="p-strip--light is-shallow" style="margin-top: -1px;">
-  <div class="row">
-    <div class="col-8 col-medium-4">
-      <p>Interested in becoming an Ubuntu partner? Talk to us today to learn more about our programmes.</p>
-    </div>
-    <div class="col-4 col-medium-2">
-      <a href="/partners/become-a-partner" class="p-button--positive">Become a partner</a>
-    </div>
-  </div>
-</section>
+{% with title="Meet some of our channel/reseller partners", link_text="See all of our channel/reseller partners ›", link_href="/partners/find-a-partner?filters=channel-reseller" %}
```

Comment:
The title is different in the design, but since we don't have updated copy docs I'm not sure which is correct 😩

Reply:
The design's copy seem to be incorrect copy and paste so I just went with what we already had!